services/output.service.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 EPAM Systems
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const vscode = require("vscode");
+
+module.exports = class OutputService {
+  static activate() {
+    if (this._outputChannel) return;
+
+    this._outputChannel = vscode.window.createOutputChannel("Alita Code");
+  }
+
+  static getChannel() {
+    if (!this._outputChannel) this.activate();
+
+    return this._outputChannel;
+  }
+
+  static logRequest(config) {
+    this._outputChannel.appendLine(`\n--- Request ---`);
+    this._outputChannel.appendLine(`Method: ${config.method.toUpperCase()}`);
```

Comment:
We may add headers and payload for requests as well, but I'm not sure it's the best choice in terms of security 🤔

Reply:
Seems like we do need.
Also, we can limit it via sepearate property `debug mode` - add interceptors ONLY when user enabled it